packages/common/src/index.ts

```diff
@@ -275,21 +291,7 @@ export type NumberFilter =
     | { operator: 'isNull'; id?: string }
     | { operator: 'notNull'; id?: string };
 
-export type DateFilterGroup = {
-    type: 'date';
-    tableName: string;
-    fieldName: string;
-    operator: FilterGroupOperator;
-    filters: DateAndTimestampFilter[];
-};
-
-export type TimestampFilterGroup = {
-    type: 'timestamp';
-    tableName: string;
-    fieldName: string;
-    operator: FilterGroupOperator;
-    filters: DateAndTimestampFilter[];
-};
+export type BooleanFilter = { operator: 'is'; value?: boolean; id?: string };
```

Comment:
I think we should use `equals` instead of `is` so it's consistent with the rest of the filters. 
We could also support the `isNull` and `notNull` for booleans since we have it for all the types.